modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/ColumnFamilyUtils.java

```diff
@@ -58,9 +64,35 @@ static ColumnFamilyType fromCfName(String cfName) {
                 return PARTITION;
             } else if (HASH_INDEX_CF_NAME.equals(cfName)) {
                 return HASH_INDEX;
+            } else if (cfName.startsWith(SORTED_INDEX_CF_PREFIX)) {
+                return SORTED_INDEX;
```

Comment:
So we have a column family for every index? Why not one cf for all indexes?

Reply:
The original idea was that you can only specify a single comparator per CF. But now I gave it a thought, and I think that we can get away with a single CF if we include the index ID in the index row and propagate a map from Index ID to its schema into the comparator. I will try to implement that.

Reply:
I've had a little discussion and we decided to keep a single CF for performance reasons (so that you don't have to lookup index schema for every comparison). I will create a ticket to test the performance of the single CF approach